ios/sdk/src/ScreenShareEventEmitter.m

```diff
@@ -14,24 +14,34 @@
  * limitations under the License.
  */
 
-#import "ScheenshareEventEmiter.h"
-#import "JitsiMeet+Private.h"
-#import "ExternalAPI.h"
+#import "ScreenShareEventEmitter.h"
+#import <React/RCTLog.h>
+#import <React/RCTBridge.h>
 
 NSNotificationName const kBroadcastStartedNotification = @"iOS_BroadcastStarted";
 NSNotificationName const kBroadcastStoppedNotification = @"iOS_BroadcastStopped";
 
-@implementation ScheenshareEventEmiter {
+static NSString * const toggleScreenShareAction = @"org.jitsi.meet.TOGGLE_SCREEN_SHARE";
+
+@implementation ScreenShareEventEmitter {
     CFNotificationCenterRef _notificationCenter;
+    bool hasListeners;
 }
 
+RCT_EXPORT_MODULE();
+
+- (NSDictionary *)constantsToExport {
```

Comment:
Please check this warning

Module ScreenShareEventEmitter requires main queue setup since it overrides `constantsToExport` but doesn't implement `requiresMainQueueSetup`. In a future release React Native will default to initializing all native modules on a background thread unless explicitly opted-out of.